stock_move_value_report/report/stock_move_line_value_report.xml

```diff
@@ -84,8 +92,23 @@
                         <tr class="border-black">
                             <td><strong>Total</strong></td>
                             <td class="text-right">
+                                <t t-set="total" t-value="0" />
+                                <t t-foreach="move_lines" t-as="move_line">
+                                    <t
+                                        t-set="valuation_layers"
+                                        t-value="move_line.move_id.stock_valuation_layer_ids"
+                                    />
+                                    <t
+                                        t-set="cost"
+                                        t-value="sum(valuation_layers.mapped(lambda svl: svl.unit_cost if svl.quantity > 0 else -svl.unit_cost))"
```

Comment:
Why don't use svl.value?

Reply:
Take a look at this:
https://github.com/odoo/odoo/blob/1379970f916302aa7b5d2f5fb134f50baa62edec/addons/stock_account/models/product.py#L137-L158

Reply:
this operation is for stock.move.lines not for stock.moves

Reply:
Ok